### ISSUE TITLE
City4CFD validates configuration when "lod" key is missing.

### ISSUE THREAD

**Issue by amorfinv** [tudelft3d/City4CFD#6]
Hello,

I noticed that City4CFD validates the configuration file when the "lod" key is missing. However, it then fails right after with a vague error message. The configuration I am using is as follows:

```json
{
  "point_clouds" :
  {
    "ground": "pointcloud/ground.txt"
  },
  "polygons" : [
    {
      "type": "Building",
      "path": "polygons/buildings.geojson",
      "unique_id": "fid",
      "height_attribute": "b3_h_max"
    },
    {
      "type": "SurfaceLayer",
      "path": "polygons/vegetation.geojson"
    },
    {
      "type":       "SurfaceLayer",
      "path":       "polygons/water.geojson"
    }
  ],

  "point_of_interest": [93898,436173],
  "influence_region":   null,
  "domain_bnd":         null,
  "top_height":         300,

  "bnd_type_bpg":      "Round",
  "bpg_blockage_ratio": false,
  "buffer_region":      20,
  "reconstruct_boundaries": true,

  "terrain_thinning": 10,
  "smooth_terrain":   {
    "iterations": 1,
    "max_pts": 100000
  },
  "building_percentile": 90,

  "edge_max_len": 5,

  "output_file_name":  "Mesh",
  "output_format":     "obj",
  "output_separately":  true,

  "output_log": true,
  "log_file": "logFile.log"
}
```

This config gives the following error:

```

     #==============================================================#
     #                        __                                    #
     #                   __  |''|                                   #
     #                  |""| |''|  _   /|__                         #
     #                __|""| |''|_| | | |""|/\_                     #
     #               |''|""| |''|'| __| |""|'''|  _____             #
     #          _ _  |''|""|^|''|'||""| |""|'''| |"""""|            #
     #         |"|"| |''|""|||''|'||""| |""|'''| |"""""|            #
     #     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~     #
     #    ____   _   _                ___________________________   #
     #   / ___| /_\ | |_   _   _      __  ____/___  ____/___  __ \  #
     #  | |     |"| | __| | | | |     _  /     __  /_    __  / / /  #
     #  | |___  |"| | |_  | |_| |  4  / /___   _  __/    _  /_/ /   #
     #   \____| |"|  \__|  \__, |     \____/   /_/       /_____/    #
     #                     |___/                                    #
     #                                                              #
     #==============================================================#
City4CFD Copyright (C) 2021-2023 3D Geoinformation Research Group, TU Delft

Work directory: "~/cfd_trial"

Validating JSON configuration file...Configuration file is valid!


Program failed! Reason: [json.exception.invalid_iterator.214] cannot get value
End
```

**Comment by ipadjen** [comment]
👋 @amorfinv. Thank you for reporting. I can recreate the issue, it is a bug in the JSON schema.

**Comment by ipadjen** [comment]
Fixed in 5e98b3e. Thanks again!
